### ISSUE TITLE
[Toolbar] Announce the suggested website in voice over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#27880]
When a user is entering a url into the address toolbar with voice over running, we should announce the suggested website.

{quote}`When typing in the address bar, voiceover screenreader does not read the suggested website it believes I am looking for. It is there, but voiceover itself isn’t reading it out. It could be helpful to type and hear when firefox thinks I am looking for so that I can either tap go or keep typing to narrow things down.`{quote}

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-12797)

**Comment by data-sync-user** [comment]
➤ Andres Furlan commented:

Duplicated from https://mozilla-hub.atlassian.net/browse/FXIOS-12872 ( https://mozilla-hub.atlassian.net/browse/FXIOS-12872|smart-link )
